listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -75,80 +75,46 @@ def __init__(self, conf, logger):
         self.dump_temp_dir_root = conf.get(
             'LISTEN_DUMP_TEMP_DIR_ROOT', tempfile.mkdtemp())
 
-    def set_empty_cache_values_for_user(self, user_name):
-        """When a user is created, set the listen_count and timestamp keys so that we
-           can avoid the expensive lookup for a brand new user."""
-
-        cache.set(REDIS_USER_LISTEN_COUNT + user_name, 0, expirein=0, encode=False)
-        cache.set(REDIS_USER_TIMESTAMPS + user_name, "0,0", expirein=0)
+    def set_empty_values_for_user(self, user_name, user_id):
+        """When a user is created, set the timestamp keys and insert an entry in the listen count
+         table so that we can avoid the expensive lookup for a brand new user."""
```

Comment:
Nit "so that we can avoid the expensive lookup for a brand new user." no longer applies since we dont use cont aggs anymore.